config.json

```diff
@@ -1552,7 +1553,7 @@
       "uuid": "8da586c3-9327-46e2-ad32-c8423061912d",
       "slug": "numbers",
       "name": "Numbers",
-      "blurb": "TODO: add blurb for numbers concept"
+      "blurb": "There are two different kinds of numbers in JavaScript - numbers and bigints. Numbers are for everyday use and are always floating point numbers."
```

Comment:
We haven't yet standardized this, but I think @iHiD suggested we use a format of: "<describe what will be learnt> followed by <describe the setting in which it will be learnt>"

This is definitely not blocking, as we have not yet documented this anywhere.